lib/writeGltf.js

```diff
@@ -1,16 +1,25 @@
 'use strict';
+var Cesium = require('cesium');
 var fsExtra = require('fs-extra');
 var path = require('path');
-var async = require('async');
-var writeSource = require('./writeSource');
-var removePipelineExtras = require('./removePipelineExtras');
-var Cesium = require('cesium');
+var Promise = require('bluebird');
+
 var defined = Cesium.defined;
 var DeveloperError = Cesium.DeveloperError;
 
+var removePipelineExtras = require('./removePipelineExtras');
+var writeSource = require('./writeSource');
+
+fsExtra.outputJson = Promise.promisify(fsExtra.outputJson);

```

Comment:
Make this something like outputJsonAsync or outputJsonPromise; don't overwrite the original function.